sdk/test-utils/identity-extensions/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@azure-tools/identity-extensions",
+  "version": "1.0.0",
+  "sdk-type": "utility",
+  "description": "Test utilities library that provides the test credential",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "browser": {
+    "./dist-esm/src/global.js": "./dist-esm/src/global.browser.js"
+  },
+  "types": "./types/latest/src/index.d.ts",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build": "npm run clean && tsc -p . && rollup -c 2>&1",
+    "build:samples": "echo Skipped.",
+    "build:test": "echo not needed",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint --no-eslintrc -c ../../.eslintrc.internal.json package.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint --no-eslintrc -c ../../.eslintrc.internal.json package.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "test:browser": "npm run clean && npm run build npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build && npm run unit-test:node",
+    "test": "npm run clean && npm run build && npm run unit-test"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/src/index.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "Azure",
+    "cloud",
+    "test",
+    "framework"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/test-utils/identity-extensions/README.md",
+  "sideEffects": false,
+  "private": true,
+  "dependencies": {
+    "@azure/core-auth": "^1.3.2",
+    "@azure-tools/test-recorder": "^1.0.0",
+    "@azure-tools/test-recorder-new": "1.0.0",
+    "@azure/identity": "^2.0.1"
```

Comment:
Identity currently uses the recorder. Wouldn’t it be an issue?

Reply:
identity depends on 
- recorder

test-utils/identity-extensions depends on
- recorder
- identity


every other SDK will depend on
- recorder
- identity
- test-utils/identity-extensions

Reply:
identity doesn't depend on test-utils/identity-extensions, so there should not be a problem